tests/test_strided_rolling.py

```diff
@@ -120,12 +121,442 @@ def test_abstract_class(dummy_data):
+        df_eda, window=1000, strides=[50], window_idx="end"
     )
     return stroll.apply_func(FuncWrapper(np.min))
+
+
+def test_sequence_stroll_indexing():
+    s = pd.Series(data=[0, 1, 2, 3, 4], name="dummy")
+
+    ## No Force
+    sr = SequenceStridedRolling(s, window=3, strides=[1], window_idx="begin")
+    assert np.all(sr.index == [0,1])
+    sr = SequenceStridedRolling(s, window=3, strides=[2], window_idx="begin")
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=3, strides=[3], window_idx="begin")
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=3, strides=[4], window_idx="begin")
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=3, strides=[5], window_idx="begin")
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=3, strides=[50], window_idx="begin")
+    assert np.all(sr.index == [0])
+
+    sr = SequenceStridedRolling(s, window=4, strides=[1], window_idx="begin")
+    assert np.all(sr.index == [0])
+
+    sr = SequenceStridedRolling(s, window=5, strides=[1], window_idx="begin")
+    assert np.all(sr.index == [])
+    sr = SequenceStridedRolling(s, window=5, strides=[2], window_idx="begin")
+    assert np.all(sr.index == [])
+    sr = SequenceStridedRolling(s, window=6, strides=[1], window_idx="begin")
+    assert np.all(sr.index == [])
+    sr = SequenceStridedRolling(s, window=6, strides=[2], window_idx="begin")
+    assert np.all(sr.index == [])
+
+    ## Force
+    sr = SequenceStridedRolling(s, window=3, strides=[1], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0,1,2])
+    sr = SequenceStridedRolling(s, window=3, strides=[2], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0,2])
+    sr = SequenceStridedRolling(s, window=3, strides=[4], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0, 4])
+    sr = SequenceStridedRolling(s, window=3, strides=[5], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=3, strides=[50], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0])
+
+    sr = SequenceStridedRolling(s, window=4, strides=[1], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0, 1])
+
+    sr = SequenceStridedRolling(s, window=5, strides=[1], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=5, strides=[2], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=6, strides=[1], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0])
+    sr = SequenceStridedRolling(s, window=6, strides=[2], window_idx="begin", include_final_window=True)
+    assert np.all(sr.index == [0])
```

Comment:
Do you agree with this behavior @jonasvdd? Now only 1 incomplete window (i.e., < window size) is included because of the flag